diff --git a/sscanApp/src/Makefile b/sscanApp/src/Makefile
index 7853dcb..012e246 100644
--- a/sscanApp/src/Makefile
+++ b/sscanApp/src/Makefile
@@ -44,15 +44,7 @@ ifdef SNCSEQ
 endif
 
 # save scan data stuff
-# XDR not available on WIN32
-ifeq ($(OS_CLASS), WIN32)
 sscan_SRCS += saveData_writeXDR.c, writeXDR.c
-else
-sscan_SRCS += xdr_lib.c
-sscan_SRCS += saveData.c
-sscan_SRCS_vxWorks += xdr_stdio.c
-sscan_SYS_LIBS_cygwin32 += $(CYGWIN_RPC_LIB)
-endif
 
 # for scanProgress
 ifdef SNCSEQ
